e2e/playwright/point-click.spec.ts

```diff
@@ -4393,4 +4533,319 @@ extrude001 = extrude(profile001, length = 1)
       )
     })
   })
+
+  const multiProfileSweepsCode = `sketch001 = startSketchOn(XY)
+profile001 = circle(sketch001, center = [3, 0], radius = 1)
+profile002 = circle(sketch001, center = [6, 0], radius = 1)
+path001 = startProfile(sketch001, at = [0, 0])
+  |> yLine(length = 2)
+  `
```

Comment:
And below here testing the other new feature: multi-profile extrude, rotate, and sweep